CODE_OF_CONDUCT.md

```diff
@@ -23,15 +23,16 @@ Ibis is governed by the
 ## Reporting and Enforcement Violations
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the Ibis Code of Conduct committee at
-ibis-conduct@googlegroups.com. You can also report
+(privately) reported by contacting the [Ibis Code of Conduct
+committee](mailto:ibis-conduct.5e76f3596b52d3bb7ca89a83e6cdbb54.show-sender@streams.zulipchat.com).
 
 The committee currently consists of:
```

Comment:
should this list be removed in favor of a link to https://github.com/ibis-project/governance/blob/main/governance.md#steering-committee-members so it doesn't need to be maintained in both places?

Reply:
Yeah, let's do that for now.  There's a future where those aren't the same set, but we aren't in it yet.

Reply:
Done